src/include/drbd/resource_conf.rb

```diff
@@ -744,6 +754,12 @@ def ResDialog(resname)
               next
           end
 
+	  if ! ValidDeviceName()
```

Comment:
Nitpicking: it is more "rubyish" if you call it without parenthesis: `if !ValidDeviceName`.

Reply:
Tried but got error, so I change it back:)

Reply:
Ah, your method name starts with uppercase letter. That confuses to ruby, because ruby expects it to be a constant.